### PR TITLE
Jakartaee wab installer

### PR DESCRIPTION
**Pull request by sebratton**

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.app.manager.wab.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+
+symbolicName = com.ibm.wsspi.appserver.app.manager.wab-1.0
+visibility = private
+-features=com.ibm.websphere.appserver.eeCompatible-8.0
```
Comment (tjwatson) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.app.manager.wab.feature:5]: This needs to tolerate 6.0 and 7.0 for eeCompatible to be able to load with servlet-3.0 and servlet-3.1

Comment (sebratton) [reply dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.app.manager.wab.feature]: Thanks. I added tolerates eeCompatible 6.0,7.0 to the feature.

**Comment by sebratton** [comment]
#build

File: dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/ConfigurableWABTests.java [hunk 2]
```diff
@@ -0,0 +1,208 @@
+/*
+ * IBM Confidential
```
Comment (tevans78) [inline dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/ConfigurableWABTests.java:2]: @sebratton please remove all IBM Confidential headers, templates and code ... ASAP!!

Comment (sebratton) [reply dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/ConfigurableWABTests.java]: Done. Thanks.

**Comment by sebratton** [comment]
#run-libby-bot

File: dev/com.ibm.ws.app.manager.wab.installer_fat/test-bundles/product1/resources/OSGI-INF/metatype/metatype.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed Materials - Property of IBM
+
+  "Restricted Materials of IBM"
```
Comment (tevans78) [inline dev/com.ibm.ws.app.manager.wab.installer_fat/test-bundles/product1/resources/OSGI-INF/metatype/metatype.xml:5]: @sebratton you missed some! Please remove ASAP!

**Comment by sebratton** [comment]
#libby-bot

**Comment by sebratton** [comment]
#libby-bot

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
Closes #12281

**Pull request merged**